e2e/test/e2e_suite_test.go

```diff
@@ -28,7 +28,7 @@ var (
 	clusterName string
 	linodeDebug = false
 	linodeURL   = "https://api.linode.com"
-	region      = "us-east"
+	region      = "us-sea"
```

Comment:
The new way needs a region where Metadata service is available. Does this change hurt?

Reply:
Should be fine, we've moved away from legacy sites for E2E testing in other projects for the most part